include/ironside/se/update.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
```

Comment:
Missing library documentation? Should be documented under doc/nrf/libraries and added to the changelog.

Reply:
Moved the implementation upstream.